openfisca_aotearoa/variables/acts/social_security/disability_allowance/disability_allowance.py

```diff
@@ -0,0 +1,353 @@
+
+    def formula_2018_11_26(persons, period, parameters):
+        meets_clause_10_conditions = persons("disability_allowance__income_limit_clause_10", period) #single person (16/17) w/out dep children
+        meets_clause_11_conditions = persons("disability_allowance__income_limit_clause_11", period) #single person w/out dep children
+        meets_clause_12_conditions = persons("disability_allowance__income_limit_clause_12", period) #in relationship
+        meets_clause_13_conditions = persons("disability_allowance__income_limit_clause_13", period) #single person with 1 child
+        meets_clause_14_conditions = persons("disability_allowance__income_limit_clause_14", period) #single person with > 1 child
+
+        return meets_clause_10_conditions + meets_clause_11_conditions + meets_clause_12_conditions 
+        + meets_clause_13_conditions + meets_clause_14_conditions
+
+
+class disability_allowance__ongoing_additional_expenses(variables.Variable):
+    value_type = bool
+    default_value = False
+    entity = entities.Person
+    definition_period = periods.WEEK
+    label = "Does the person have additional expenses of an ongoing kind arising from the person’s disability?"
+    reference = "https://www.legislation.govt.nz/act/public/2018/0032/latest/whole.html#DLM6783280"
+
+
+
+class disability_allowance__person_entitled_to_reciprocal_benefits(variables.Variable):
+    value_type = bool
+    default_value = False
+    entity = entities.Person
+    definition_period = periods.WEEK
+    label = "Reciprocal benefits for person entitled to reciprocal benefits"
+    reference = "https://www.legislation.govt.nz/act/public/2018/0032/latest/whole.html#DLM6783280"
+
+
+class disability_allowance__receiving_any_other_disability_allowance(variables.Variable):
+    value_type = bool
+    default_value = False
+    entity = entities.Person
+    definition_period = periods.WEEK
+    label = "Reciprocal benefits for person entitled to reciprocal benefits"
+    reference = "https://www.legislation.govt.nz/act/public/2018/0032/latest/whole.html#DLM6783280"
+
+
+class disability_allowance__receiving_accident_compensation_entitlement(variables.Variable):
+    value_type = bool
+    default_value = False
+    entity = entities.Person
+    definition_period = periods.WEEK
+    label = "Is the person receiving accident compensation entitlement?"
+    reference = "https://www.legislation.govt.nz/act/public/2018/0032/latest/whole.html#DLM6783280"
+
+
+class disability_allowance__receiving_disablement_pension(variables.Variable):
+    value_type = bool
+    default_value = False
+    entity = entities.Person
+    definition_period = periods.WEEK
+    label = "Is the person receiving a disablement pension under Part 3?"
+    reference = "https://www.legislation.govt.nz/act/public/2018/0032/latest/whole.html#DLM6783280"
+
+
+class disability_allowance__receiving_veterans_support_entitlement(variables.Variable):
```

Comment:
rename as discussed ( __receiving)

Reply:
removed this variable as it wasn't needed!